rdr_service/etl/raw_sql/partially_initialize_cdm.sql

```diff
@@ -522,3 +522,60 @@ CREATE TABLE cdm.dose_era
     PRIMARY KEY (dose_era_id),
     UNIQUE KEY (id)
 );
+
+-- -----------------------------------------------
+-- create and populate tmp_questionnaire_response
+-- -----------------------------------------------
+DROP TABLE IF EXISTS cdm.tmp_questionnaire_response;
+CREATE TABLE cdm.tmp_questionnaire_response (
+      questionnaire_response_id int(11),
+      participant_id int(11),
+      questionnaire_id int(11),
+      created DATETIME,
+      authored DATETIME,
+      identifier varchar(80),
+      answers_hash VARCHAR(80),
+      duplicate int(11),
+      removed int(11)
+);
+SET TRANSACTION ISOLATION LEVEL READ UNCOMMITTED;
+INSERT INTO cdm.tmp_questionnaire_response (
+   questionnaire_response_id, participant_id, questionnaire_id, created, authored, identifier, duplicate, removed)
+SELECT questionnaire_response_id, participant_id, questionnaire_id, created, authored,
+       REPLACE((JSON_EXTRACT(CONVERT(resource USING utf8 ), '$.identifier.value')), '"', '') AS identifier,
+       0 AS duplicate, 0 AS removed
+FROM rdr.questionnaire_response qr;
+CREATE UNIQUE INDEX uidx_qr_lookup ON cdm.tmp_questionnaire_response (questionnaire_response_id);
+drop temporary table if exists cdm.answer_hash_values;
+create temporary table cdm.answer_hash_values (
+      questionnaire_response_id int(11),
+      answers_hash VARCHAR(80)
+);
+explain
```

Comment:
I accidentally left in an `explain` when I copied this over. Sorry.

Reply:
Removed it.